### PR TITLE
OIDC Compliance Test Suite

### PR DESCRIPTION
**Pull request by NipuniBhagya**

### CODE REVIEW

File: modules/samples/oauth2/OIDC-Test-Suite/src/app/components/OauthClient.js [hunk 1]
```diff
@@ -0,0 +1,888 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (nilasini) [inline modules/samples/oauth2/OIDC-Test-Suite/src/app/components/OauthClient.js]: Change the year in all the places

File: modules/samples/oauth2/OIDC-Test-Suite/src/CSS/index.css [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+header {
+    background-color: #000000;
+    padding: 10px;
+    box-shadow: 0 1px 3px rgba(0,0,0,0.12), 0 1px 2px rgba(0,0,0,0.24);
+    color: #CECECE;
+    font-size: 120%;
+    width:100%;
+    height: 60px;
+
```
Comment (vihanga-liyanage) [inline modules/samples/oauth2/OIDC-Test-Suite/src/CSS/index.css]: Can we remove these spaces?

File: modules/samples/oauth2/OIDC-Test-Suite/src/CSS/index.css [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    left: 0;
+    bottom: 0;
+    background-color: #242323;
+    padding: 14px;
+    box-shadow: 0 1px 3px rgba(0,0,0,0.12), 0 1px 2px rgba(0,0,0,0.24);
+    color: #CECECE;
+    width:100%;
+    height: 45px;
+    font-size: 15px;
+
```
Comment (vihanga-liyanage) [inline modules/samples/oauth2/OIDC-Test-Suite/src/CSS/index.css]: Can we remove this additional new line?

File: modules/samples/oauth2/OIDC-Test-Suite/src/Images/logo-inverse.svg [hunk 4]
```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+ * Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (vihanga-liyanage) [inline modules/samples/oauth2/OIDC-Test-Suite/src/Images/logo-inverse.svg]: Please update the year.

File: modules/samples/oauth2/OIDC-Test-Suite/src/app/components/Authorized.js [hunk 5]
```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (vihanga-liyanage) [inline modules/samples/oauth2/OIDC-Test-Suite/src/app/components/Authorized.js]: Please update the year in all other places as well.

File: modules/samples/oauth2/OIDC-Test-Suite/src/app/components/FirstForm.js [hunk 6]
```diff
@@ -0,0 +1,382 @@
...
+        if (this.state.pro_type == "Basic") {
+            var load = 'response_type=' + response_type + '&client_id=' + clientId + '&scope=' + scope + '&redirect_uri=' + callBack + '&Use_pkce=no';
+            window.location.href = "https://localhost:9443/oauth2/authorize?" + load;
+
+        } else {
+            var data = 'response_type=' + response_type + '&client_id=' + clientId + '&redirect_uri=' + callBack + '&nonce=' + nonce + '&scope=' + scope;
+            var url = authEndpoint + "?" + data;
+            sessionStorage.requestUrl = url;
+            window.location.href = "https://localhost:9443/oauth2/authorize?" + data;
+
```
Comment (vihanga-liyanage) [inline modules/samples/oauth2/OIDC-Test-Suite/src/app/components/FirstForm.js]: Can we remove these additional new lines?

File: modules/samples/oauth2/OIDC-Test-Suite/src/app/components/OauthClient.js [hunk 7]
```diff
@@ -0,0 +1,888 @@
...
+            resultState = <h4><span className="label label-success">Success!</span></h4>;
+            this.setState({resultState});
+
+            stateText = <font color="green"> Success!!!</font>
+            this.setState({stateText});
+
+            summary = status + " " + status_Text;
+            this.setState({summary})
+        }
+        ;
```
Comment (vihanga-liyanage) [inline modules/samples/oauth2/OIDC-Test-Suite/src/app/components/OauthClient.js]: Is this semicolon required?

File: modules/samples/oauth2/OIDC-Test-Suite/src/app/components/OauthClient.js [hunk 8]
```diff
@@ -0,0 +1,888 @@
...
+                            <p>
+                                <Highlight className='javascript'>
+                                    {JSON.stringify(idEn, null, '\t')}
+                                </Highlight>
+                            </p>
+                            <br/>
+                        </div>
+                    </div>
+                </div>
+
```
Comment (vihanga-liyanage) [inline modules/samples/oauth2/OIDC-Test-Suite/src/app/components/OauthClient.js]: Please remove the additional new line.

**Comment by pulasthi7** [comment]
Hi @NipuniBhagya ,

Can you get the requested changes done to get this PR merged?

**Comment by pulasthi7** [comment]
@thanujalk @NipuniBhagya ,

Please send this to the master branch as 5.5.x is no longer used.

**Pull request merged**